subsys/fs/Kconfig.fatfs

```diff
@@ -230,6 +230,27 @@ config FS_FATFS_REENTRANT
 	  access for each volume. Will create a zephyr mutex object for each
 	  FatFs volume and a FatFs system mutex.
 
+config FS_FATFS_LBA64
+	bool "Support for 64-bit LBA"
+	depends on FS_FATFS_EXFAT
+	help
+	  This option enables support for 64-bit LBA, which also
+	  enables GPT support.
+
+config FS_FATFS_MULTI_PARTITION
+	bool "Support for multiple volumes on the physical drive"
+	help
+	  When this function is enabled, each logical drive number can
+	  be bound to arbitrary physical drive and partition listed
+	  in the VolToPart[] of the fatfs module. The mount points
```

Comment:
This is not clear from the description that this array has to be defined and provided by user. Just enabling the `FS_FATFS_MULTI_PARTITION` option will cause linker to fail to find the definition

Reply:
Hi @de-nordic,

Would below change be ok with you:
```
--- a/subsys/fs/Kconfig.fatfs
+++ b/subsys/fs/Kconfig.fatfs
@@ -242,8 +242,9 @@ config FS_FATFS_MULTI_PARTITION
        help
          When this function is enabled, each logical drive number can
          be bound to arbitrary physical drive and partition listed
-         in the VolToPart[] of the fatfs module. The mount points
-         have to be numbered in this case.
+         in the VolToPart[] of the fatfs module. The VolToPart[] is expected to be
+         provided by Zephyr application.
+         The mount points have to be numbered in this case.
          For example, 2 FAT partition on SD disk (3) in terms of Zephyr:
            {3, 1} - mount point "/0:"
            {3, 2} - mount point "/1:"
```

Reply:
Yeah. Would it be possible to provide test for this, serving also as a reference design?

Reply:
Sorry, I'm not sure as it's required specific storage format and overall configuration is a bit specific.
How about adding code example in commit message:
```
Example of mounting a second FATFS partition places on SD-card:

static FATFS fat_fs;

static struct fs_mount_t mp = {
	.type = FS_FATFS,
	.fs_data = &fat_fs,
       .mnt_point = "/1:
};

/* 
 * 2 FAT partition on SD disk
 * PARTITION.pd - Physical drive number. "SD" has index 3 in terms of Zephyr (see FF_VOLUME_STRS)
 * PARTITION.pt - Partition (0:Auto detect, 1-4:Forced partition). So 1 for the first FATFS partition and 2 - for second.
 */
PARTITION VolToPart[FF_VOLUMES] = {
    [0] = {3, 1},     /* "0:" ==> 1st partition on the pd#0 */
    [1] = {3, 2},     /* "1:" ==> 2nd partition on the pd#0 */
    [2] = {0, -1},     /* "2:" ==> 3rd partition on the pd#0 */
    [3] = {0, -1},
    [4] = {0, -1},
    [5] = {0, -1},
    [6] = {0, -1},
    [7] = {0, -1},
};

fs_mount(&mp);
```

Reply:
Seems that we would have to write MBR to flash partition, from within the test or sample.